pkg/storaged/block/create-pages.jsx

```diff
@@ -85,8 +86,14 @@ export function make_block_page(parent, block, card) {
             // can not happen unless there is a bug in the code above.
             console.error("Assertion failure: is_crypto == false");
         }
-        if (fstab_config.length > 0 && !is_btrfs) {
-            card = make_filesystem_card(card, block, null, fstab_config);
+        if (fstab_config.length > 0) {
+            if (is_btrfs) {
+                if (single_device_volume)
+                    card = make_btrfs_filesystem_card(card, block, null);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20070-20240305-074754-13ce2d6f-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)